examples/fluent-bit/grafana-cloud/README.md

```diff
@@ -0,0 +1,24 @@
+### FireLens Example: Logging to Grafana Loki with Fluent Bit
+
+For documentation on sending your FireLens monitored log data to Grafana Loki, see: [Fluent Bit loki Plugin Documentation](https://docs.fluentbit.io/manual/v/1.9-pre/pipeline/outputs/loki). Be aware there is a separate Golang output plugin provided by [Grafana](https://grafana.com/docs/loki/latest/clients/fluentbit/) with different configuration options. The Loki data source provides access to Loki, Grafana’s log aggregation system.
```

Comment:
I think we should also link this blog, it seems useful: https://calyptia.com/blog/how-to-send-logs-to-loki-using-fluent-bit

We're already linking third party company docs here, so I think its fine. 

We can call it out something like, "Here is a [blog](https://calyptia.com/blog/how-to-send-logs-to-loki-using-fluent-bit) on sending logs to Loki using the `loki` Fluent Bit plugin."